### PR TITLE
#22 all events page

### PR DESCRIPTION
**Pull request by Shubham2301**
PR for #22

### CODE REVIEW

**Comment by Shubham2301** [comment]
access test page from 
http://ec2-52-14-210-239.us-east-2.compute.amazonaws.com/all-soirees/

**Comment by rathorevaibhav** [comment]
@Shubham2301 Please fix. Cannot open Soiree from "all soiree" page. The links don't take anywhere.

**Comment by rathorevaibhav** [comment]
Something's not right on UI, I am able to scroll horizontally.
image


See the whitespace right to the navbar.

Also, unable to open July Soiree. It should be a link. @Shubham2301 Please fix them before I further test.

File: public/wp-content/themes/ColoredCow/all_soiree.php [hunk 1]
```diff
@@ -12,35 +12,33 @@
...
+			$last_date = get_field('last_date');
+			$current_date = date("Y-m-d");
 	?>			
 				<div class="container">
 					<div class="row">
-						<div class="col-md-12">	
-							<h1 class="page-title text-center"><a href="<?php if($current_date <= $last_date){echo get_home_url();}?>"><?php echo get_the_title();?></a></h1>
+						<div class="col-xs-12">	
+							<h1 class="page-title text-center"><?php if($current_date <= $last_date):?><a href="<?php echo get_home_url();?>"
+						<?php echo ">"; endif;?><?php echo get_the_title();?></a></h1>
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/all_soiree.php]: put echo ">" out of PHP tags?

File: public/wp-content/themes/ColoredCow/all_soiree.php [hunk 2]
```diff
@@ -12,35 +12,33 @@
...
-			$current_date = strtotime(date("d/m/y"));
+			$last_date = get_field('last_date');
+			$current_date = date("Y-m-d");
 	?>			
 				<div class="container">
 					<div class="row">
-						<div class="col-md-12">	
-							<h1 class="page-title text-center"><a href="<?php if($current_date <= $last_date){echo get_home_url();}?>"><?php echo get_the_title();?></a></h1>
+						<div class="col-xs-12">	
+							<h1 class="page-title text-center"><?php if($current_date <= $last_date):?><a href="<?php echo get_home_url();?>"
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/all_soiree.php]: breaking this line into multiple might help to make the code clean and non-confusing.

File: public/wp-content/themes/ColoredCow/header.php [hunk 3]
```diff
@@ -9,13 +9,17 @@
...
-        <!-- <div>
-            <ul class = "nav navbar-nav navbar-right" style= "margin-right: 0">
-                <li class = ""><a href = "#">RSVP</a></li>
+        <div>
+            <ul class = "nav navbar-nav navbar-right element" style= "margin-right: 0">
+                <?php 
+                    $page = get_page_by_title("All Soirees");
+                    $link_all_soiree_page = get_page_link($page->ID) 
+                ?>
+                <li class = ""><a href = "<?php echo $link_all_soiree_page; ?>">All-Soirees</a></li>
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/header.php]: class="" needed? please remove if not.

**Review by rathorevaibhav — CHANGES_REQUESTED** [review]
just some basic changes. else everything is fine.

**Comment by rathorevaibhav** [comment]
Looks good on testing. There's a bug that I've reference is issue #32. Let's track that later.

Please work on the review feedback and merge once done.

**Pull request merged**